src/screens/Forum/ForumPost.js

```diff
@@ -72,7 +74,7 @@ export default function ForumPost({
 
   const [visible, setVisible] = useState(false);
   return (
-    <TouchableOpacity onPress={navigateToPostScreen}>
+    <TouchableOpacity onPress={navigateToPostScreen(postID, userID)}>
```

Comment:
Its very close. `onPress` needs receive a function, but this is actually calling the function and giving whatever it returns to `onPress`. We just need to do `onPress={() => navigateToPostScreen(postID, userID)}` and it should be good!